common/djangoapps/config_models/models.py

```diff
@@ -41,7 +41,10 @@ def current_set(self):
         """
         assert self.model.KEY_FIELDS != (), "Just use model.current() if there are no KEY_FIELDS"
         return self.get_queryset().extra(           # pylint: disable=no-member
-            where=["id IN ({subquery})".format(subquery=self._current_ids_subquery())],
+            where=["{table_name}.id IN ({subquery})".format(
```

Comment:
It actually only occurs twice, and the `subquery` is different between the two. I feel it may be overkill in this case to factor this out into a separate method as such a change would result in more code, not less.

Do you think it makes sense to leave it as-is?